src/PackageEnvironment.js

```diff
@@ -102,7 +102,7 @@ function relativeToSandbox(realFromPath, toPath) {
     ? realToPath
     : toPath;
   let ret = path.relative(realFromPath, toPathToUse);
-  return (ret == '0') ? "$esy__sandbox" : path.join("$esy__sandbox", ret);
+  return (ret == '0') ? "$esy__sandbox" : `$esy__sandbox/${ret}`;
```

Comment:
Isn't `path.join` better since it will be more compatible with windows (uses the correct slash)? I know we shouldn't *worry* about windows, but perhaps the more closer we are the better? Or does an msys approach make it better to use `/`?